gem/gem.py

```diff
@@ -553,6 +557,13 @@ def __init__(self, variable, dim2idxs):
         self.dim2idxs = tuple(dim2idxs_)
         self.free_indices = unique(free_indices)
 
+    def index_ordering(self):
+        """Running indices in the order of indexing in this node."""
+        return tuple(index
+                     for offset, idxs in self.dim2idxs
+                     for index, stride in idxs
```

Comment:
Could write `for _, ids in self.dim2idxs for index, _ in idxs` so it's clear that those variables aren't being used?